app/assets/javascripts/views/map_file_categories_view.js

```diff
@@ -0,0 +1,164 @@
+'use strict';
+
+(function(App) {
+
+  App.View = App.View || {};
+
+  App.View.MapFileCategories = Backbone.View.extend({
+
+    defaults: {
+      ignored_categories: [
+        'cartodb_id',
+        'the_geom',
+        'the_geom_webmercator',
+        'updated_at',
+        'created_at',
+        'year'
+      ]
+    },
+
+    columns: [],
+
+    events: {
+
+    },
+
+    initialize: function(params) {
+      this.options = _.extend({}, this.defaults, params.options || {});
+      this.data = this._getAppData();
+      this.ignored_categories = this.options.ignored_categories;
+      this.columnsContainer = this.el.getElementsByClassName('box-list')[0];
+      this.customColumsInput = this.el.getElementsByClassName('custom_columns_colors')[0];
+      this.palette = App.CartoCSS['Theme' + this.data.caseStudy.template].palette1;
+    },
+
+    init: function(column) {
+      var self = this
+      var table = this.el.getAttribute('data-table');
+      column = column || this.el.getAttribute('data-column');
+      this.columnsContainer.classList.add('_is-loading');
+      this.columnsContainer.innerHTML = '';
+      if (table && column) {
+        var promise = self.getCategories(table, column);
+        promise.done(function(categories){
+          self.refreshCategories(categories);
+        });
+        promise.fail(function(error){
+          self.handleCategoriesError(error);
+        });
+      } else {
+        this.columnsContainer.innerHTML = 'Please choose a table column';
+        this.columnsContainer.classList.remove('_is-loading');
+      }
+    },
+
+    initColorPicker: function() {
+      var self = this;
+      $('.colorpicker').each(function(index, item){
+        var $item = $(item),
+            borderColor;
+        $item.spectrum({
+		      showAlpha: true,
+          showInput: true,
+          showInitial: true,
+          showPalette: true,
+          showSelectionPalette: true,
+          maxSelectionSize: 10,
+          preferredFormat: 'rgb',
+          palette: self.palette
+        });
+        borderColor = $(item).spectrum('get').toHex();
+        $(item).siblings('.sp-replacer').css('border','1px solid #'+borderColor);
+
+        $(item).on('change.spectrum', function(e, color){
+          var borderContainer = e.currentTarget.parentElement.getElementsByClassName('sp-replacer')[0];
+          borderContainer.style.border = '1px solid #'+color.toHex();
+          self.updateColumnsColor(color);
+        });
+      });
+
+    },
+
+    getCategories: function(table, column) {
+      var query = 'SELECT DISTINCT ' + column +' AS CATEGORY FROM ' + table + ' LIMIT 15';
+      var defer = new $.Deferred();
+      $.getJSON('https://'+this.data.cartodb_user+'.cartodb.com/api/v2/sql/?q='+query)
```

Comment:
I saw that CartoDB has an example with getJSON, but just so you know CartoDB.js also has a method to make SQL queries http://docs.cartodb.com/cartodb-platform/cartodb-js/sql/#arguments